theano/gof/tests/test_opt.py

```diff
@@ -572,6 +572,8 @@ def test_2(self):
         assert str(g) == '[Op2(x, y)]'
 
     def test_low_use_ratio(self):
+        backup = theano.config.on_opt_error
+        theano.config.on_opt_error = 'warn'
```

Comment:
@nouiz: This test was made explicitly so that max_use_ratio is small and the condition is triggered.
It used to test that this was not a failure (and silences the error message) and that the optimization process would move on.
Now that this is actually a failure if on_opt_error is "raise", I think we want to change the value of on_opt_error for that test, which is what I asked Reyhane to do.

@ReyhaneAskari I think "ignore" would actually be a better value for that test. There is also a decorator you can use instead of using the existing try / finally block: `@theano.configparser.change_flags(on_opt_error=ignore)`.